docs/current/162770-faq.md

```diff
@@ -2,12 +2,119 @@
+
+### How do I update Dagger?
+
+:::tip
+[Learn more about compatibility between the Dagger Engine, the Dagger CLI and Dagger SDKs](#what-compatibility-is-there-between-the-dagger-engine-dagger-sdks-and-dagger-cli-versions).
+:::
+
+#### CLI
+
+To update the Dagger CLI, download and uncompress the [latest release of the Dagger CLI](https://github.com/dagger/dagger/releases) for your platform (replace the placeholders with the correct values) and replace your existing Dagger CLI binary with the newer version:
+
+```shell
+wget https://github.com/dagger/dagger/releases/download/vMAJOR.MINOR.PATCH/dagger_vMAJOR.MINOR.PATCH_PLATFORM_ARCHITECTURE.tar.gz
+tar -xzvf dagger_vMAJOR.MINOR.PATCH_PLATFORM_ARCHITECTURE.tar.gz
+sudo mv dagger /usr/local/bin/dagger
+```
+
+Homebrew users can alternatively use the following commands:
+
+```shell
+brew update
+brew upgrade dagger
+```
+
+#### SDKs
+
+To update a Dagger SDK, follow the same procedure that you would follow to update any other SDK package in your chosen development environment. For example:
+
+- Go SDK
+
+  ```shell
+  go get dagger.io/dagger@latest
+  ```
+
+- Node.js SDK
+
+  <Tabs>
+  <TabItem value="npm">
+
+  ```shell
+  npm install @dagger.io/dagger@latest --save-dev
+  ```
+
+  </TabItem>
+
+  <TabItem value="yarn">
+
+  ```shell
+  yarn upgrade --latest @dagger.io/dagger --dev
+  ```
+
+  </TabItem>
+  </Tabs>
+
+- Python SDK
+
+  ```shell
+  pip install -U dagger-io
+  ```
+
+  If using Conda:
+
+  ```shell
+  conda update dagger-io
+  ```
+
+### What compatibility is there between the Dagger Engine, Dagger SDKs and Dagger CLI versions?
+
+- The Dagger CLI is released in tandem with the Dagger Engine and thus shares a version number with it.
+- Dagger SDKs automatically provision a Dagger Engine at a compatible version.
+
+Each release notes the compatible Dagger Engine version in its release notes. If running from the CLI, or providing a CLI for the SDK to use via the `_EXPERIMENTAL_DAGGER_CLI_BIN` variable, check the release notes of the SDK, which indicate the required CLI and Engine versions.
+
+The image below shows an example of the compatibility information available in the release notes:
+
+![Release information](/img/current/faq/release-notes.png)
+
+### How do I uninstall Dagger?
+
+Follow these steps:
+
+1. Remove the Dagger CLI using the following command:
+
+  ```shell
+  sudo rm /usr/local/bin/dagger
+  ```
+
+  Homebrew users can alternatively use the following command:
+
+  ```shell
+  brew uninstall dagger
+  ```
+
+1. Remove the Dagger container using the following commands:
+
+  ```shell
+  DAGGER_CONTAINER_ID=`docker ps -q --filter "name=^dagger-engine-*"`
+  docker update --restart=no DAGGER_CONTAINER_ID
+  docker rm DAGGER_CONTAINER_ID
```

Comment:
I would simplify these 3 lines to:

```
docker rm --force --volumes "$(docker ps --quiet --filter='name=^dagger-engine-')"
```

I find the long flag names easier to understand (they are more descriptive). If we prefer brevity, this will achieve the same outcome:

```console
docker rm -fv "$(docker ps -qf='name=^dagger-engine-')"
```

Reply:
We've had the uninstall on the backlog for awhile, but not sure how easy it is to add to this PR. Either way, I have it on the linear docs backlog already, so I know Vikram is working on adding it soon https://github.com/dagger/dagger/issues/2123

Reply:
@mircubed is that just instructions or an actual `dagger uninstall` command like @gerhard is wishing for?

Reply:
@helderco just the instructions for now. Sorry for the confusion. I just made an issue to track the command request https://github.com/dagger/dagger/issues/5559

Reply:
Modified content